xml/System.Text/UnicodeEncoding.xml

```diff
@@ -1903,9 +1903,37 @@
         <ReturnType>System.ReadOnlySpan&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a Unicode byte order mark encoded in UTF-16 format, if this object is configured to supply one.</summary>
+        <value>A span of bytes containing the Unicode byte order mark, if this object is configured to supply one. Otherwise, the default span.</value>
```

Comment:
```suggestion
        <value>A byte span containing the Unicode byte order mark, if this object is configured to supply one; otherwise, the default span.</value>
```